wavefront_sdk/entities/histogram/histogram_impl.py

```diff
@@ -155,17 +155,16 @@ def bulk_update_dist_by_thread_id(self, thread_id, means, counts):
     def get_centroids(self):
         """Get list of centroids for dists of all threads in this minute."""
         centroids = []
-        for thread_id in self.per_thread_dist:
-            centroids.extend(
-                self.per_thread_dist[thread_id].centroids_to_list())
+        for _, digest in self.per_thread_dist.items():
```

Comment:
Please consider using the `.values()` view instead of `.items()` if you don't use keys